### PR TITLE
React react1 week1

### PR DESCRIPTION
**Pull request by andrii1**

### CODE REVIEW

**Comment by dpfernandes** [comment]
Can you please write a descriptive title for this PR as it's requested on the homework?

**Comment by andrii1** [comment]
> Can you please write a descriptive title for this PR as it's requested on the homework?

changed!

File: react/react1/week1/todolist/src/App.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import "./App.css";
+import Table from "./components/Table";
+
+function App() {
+  return (
+    <div className="App">
+      <Table />
```
Comment (annamourasobral) [inline react/react1/week1/todolist/src/App.js:7]: there's no table, so I would recommend you change the name to todo list instead

File: react/react1/week1/todolist/src/components/ItemDeadline.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+function ItemDeadline({ deadline }) {
```
Comment (annamourasobral) [inline react/react1/week1/todolist/src/components/ItemDeadline.js:1]: over nesting

File: react/react1/week1/todolist/src/components/ItemDescription.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+function ItemDescription({ description }) {
```
Comment (annamourasobral) [inline react/react1/week1/todolist/src/components/ItemDescription.js:1]: over nesting

File: react/react1/week1/todolist/src/components/Table.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
+import ItemRow from "./ItemRow";
+import Header from "./Header";
+const items = [
```
Comment (annamourasobral) [inline react/react1/week1/todolist/src/components/Table.js:3]: separate in another file

File: react/react1/week1/todolist/src/components/Table.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    deadline: "2022-10-25",
+  },
+];
+
+function Table() {
+  return (
+    <>
+      <Header />
+      {items.map(({ description, deadline, id }) => (
+        <ItemRow description={description} deadline={deadline} key={id} />
```
Comment (annamourasobral) [inline react/react1/week1/todolist/src/components/Table.js:26]: these props should only be accessed inside the item

**Review by annamourasobral — CHANGES_REQUESTED** [review]
Hey, @andrii1 
Since it's the same app some of my requests are also in PR #22.
Don't forget to style it.
Happy coding! 👩🏻‍💻

**Pull request merged**